SimTracker/Common/interface/SiPixelChargeReweightingAlgorithm.h

```diff
@@ -3,8 +3,8 @@
 
 // Original Author Caroline Collard
 // September 2020 : Extraction of the code for cluster charge reweighting from SiPixelDigitizerAlgorithm to a new class
+// Jan 2023: Generalize so it can be used for Phase-2 IT as well (Tamas Vami)
```

Comment:
the overall approach (including https://github.com/cms-sw/cmssw/pull/40349) looks reminescent of https://github.com/mmusich/cmssw/commit/f1b1ff5453edc1125a35fdc4f73040fa06385cd2

Reply:
yes, I'll add your name, thanks!